src/styles/blocks/card.scss

```diff
@@ -0,0 +1,63 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    display: block;
+    margin: 0 3px 40px;
+  }
+
+  &__title {
+    color: $main-text-color;
```

Comment:
The variable `$main-text-color` is used but not defined in this file. Ensure that it is defined or imported from another SCSS file to avoid compilation errors.